docs/COREDUMP_PACKAGE_METADATA.md

```diff
@@ -58,6 +61,7 @@ Value: a JSON string with the structure described below
      "name":"coreutils",
      "version": "4711.0815.fc13.arm32",
      "osCpe":               # A CPE name for the operating system, `CPE_NAME` from os-release is a good default
+     "debuginfod": "https://debuginfod.fedoraproject.org"
```

Comment:
no `/` suffix in the URL?

shouldn#t the field be `debuginfoURL` or even `debugInfoURL` or so? (or `debugInfoUrl`? i.e. "debuginfod" is a project name to me, but we don't want to refer to that, but to the link where to find the stuff

Reply:
fixed as requested